### ISSUE TITLE
Failure to read certain image file with new PNG Decoder

### ISSUE THREAD

**Issue by Kenny-Hui** [leezer3/OpenBVE#942]
### Description

In the latest Nightly Build with the new PNG Decoder, some PNG file will fail to load.

![image](https://github.com/leezer3/OpenBVE/assets/28094366/fbd4edf6-e838-4e34-870a-ad175430dccb)
![image](https://github.com/leezer3/OpenBVE/assets/28094366/f80074e7-a183-46ad-9459-0f9ff8a6929d)

### Reproduction

If the issue occurs in multiple routes/ trains, please provide one or two samples.
In order to reproduce the issue and debug it, it's helpful to have the following:

## Train
[HKTSS.MTR.Light.Rail.Phase.IV.v2.0.1.zip](https://github.com/HKTSS/LRV-P4/releases/download/2.0.1/HKTSS.MTR.Light.Rail.Phase.IV.v2.0.1.zip)

## Logs

[log.txt](https://github.com/leezer3/OpenBVE/files/12600243/log.txt)

### Related information 

* Operating system: Debian Testing (Trixie)
* Method of control: Keyboard

**Comment by leezer3** [comment]
Meh.
Sure I committed that, but obviously not :(

At least that was a simple one.....

TLDR:
There's one other actual public plugin for loading textures (the Chinese encryption one), so can't just drop support for it.
